### PR TITLE
Bookstore: Style

### PR DESCRIPTION
**Pull request by maov19**
## Added features
- Added missing JSX elements (without adding extra functionality)
- Final design matches Zeplin's design (this includes all elements + the same styling)
## Demo
bookstore-styling

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -55,31 +77,43 @@ function CreateBook() {
...
+            <label htmlFor="author">
+              <input
+                type="text"
+                id="author"
+                value={author}
+                onChange={(event) => setAuthor(event.target.value)}
+                placeholder="Author"
+              />
+            </label>
+          </div> */}
```
Comment (Munsa1) [inline src/components/Books.js]: - [x] KIndly remove commented code as it makes your codebase not clean

File: src/styles/booksStyles.css [hunk 2]
```diff
@@ -0,0 +1,211 @@
+.book-container {
+  margin: 0 6rem 0 6rem;
+  display: flex;
```
Comment (Munsa1) [inline src/styles/booksStyles.css]: - [ ] You could consider using a reusable class for the `flex` property as it reoccurs many times in your codebase

File: src/styles/booksStyles.css [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+.book-info {
+  width: 35%;
+  margin: 0 0 0 2rem;
+  padding: 0;
+}
+
+.book-category {
+  margin: 1.5rem 0 0.5rem 0;
+  font-family: "Montserrat", sans-serif;
```
Comment (Munsa1) [inline src/styles/booksStyles.css]: - [ ] You could consider using a reusable class for the `font-family` property as it reoccurs many times in your codebase  for CSS best practices

File: src/styles/booksStyles.css [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+  right: -1rem;
+  width: 1px;
+  height: 80%;
+  transform: translateY(-50%);
+  background-color: rgba(121, 114, 114, 0.219);
+}
+
+.completed {
+  margin: 0;
+  padding: 0;
```
Comment (Munsa1) [inline src/styles/booksStyles.css]: - [ ] You could consider using a reusable class for the `margin` and `padding`property as it reoccurs many times in your codebase as well for CSS best practices

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @maov19, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Neat ReadMe File :heavy_check_mark: 
Nice Styling :heavy_check_mark: 
Seperated Styling as Required :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (maov19) [reply src/components/Books.js]: Removed! ✔️

File: src/styles/booksStyles.css (hunk 2 above)
Comment (maov19) [reply src/styles/booksStyles.css]: Ok, thanks for the suggestion! I created the `.flex` class ✔️

File: src/styles/booksStyles.css (hunk 3 above)
Comment (maov19) [reply src/styles/booksStyles.css]: Ok, thanks for the suggestion! I created the `.montserrat` class ✔️

File: src/styles/booksStyles.css (hunk 4 above)
Comment (maov19) [reply src/styles/booksStyles.css]: Thanks! Created the `.reset` class ✔️

**Review by binhussen — APPROVED** [review]
Hi @maov19 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**